post/urls.py

```
@@ -0,0 +1,15 @@
+from django.urls import path
+from .views import PostListView, PostDetailView
+from .views import PostListView, PostDetailView, LikeView
```

Ruff (0.8.2)

Redefinition of unused `PostListView` from line 2

Remove definition: `PostListView`

(F811)